### PR TITLE
Ubiquity benchmarks

### PR DESCRIPTION
**Pull request by jcheron**
just a Pull request for [Ubiquity framework](https://ubiquity.kobject.net) (PHP).

All verifications are ok, and so are the Travis tests.
The `README.md` File has been updated.

Congratulations on the very clear procedure for setting up the tests.

Is there a period set for the round 18?

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
@jcheron Thanks, and welcome to TFB! This all looks good, the one thing I'd ask is if any non-essential files can be removed, like `public/assets/*`

File: frameworks/PHP/ubiquity/app/controllers/Db.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ **/
+class Db extends Controller{
+	public function initialize(){
+		\header( 'Content-Type: application/json;charset=utf-8' );
+		$config=Startup::getConfig();
+		\Ubiquity\orm\DAO::startDatabase($config);
+	}
+
+	public function index() {
+		$world=DAO::getOne(World::class, mt_rand(1, 2),false);
```
Comment (joanhey) [inline frameworks/PHP/ubiquity/app/controllers/Db.php]: mt_rand(1, 10000)

Comment (jcheron) [reply frameworks/PHP/ubiquity/app/controllers/Db.php]: Thanks!

**Comment by jcheron** [comment]
Thank you for your answer, @nbrady-techempower .
It's okay for the unnecessary `public` directory.

File: frameworks/PHP/ubiquity/app/controllers/Db.php (hunk 1 above)
Comment (nbrady-techempower) [reply frameworks/PHP/ubiquity/app/controllers/Db.php]: Nice catch!

**Pull request merged**